hybrid/src/main/java/jetbrains/jetpad/hybrid/ParsingHybridProperty.java

```diff
@@ -58,6 +58,7 @@ public ParsingHybridProperty(
     myPrinter = printer;
     mySourceTokens = tokens;
     myParsingContextFactory = parsingContextFactory;
+    initUpdate();
     myValue = parse();
     update();
```

Comment:
No, we need to populate first pretty comments with source comments. Otherwise, the comment will not be there. You can comment this call and the test added in this commit will fail.